test/DatabaseIntegrationTest/DatabaseIntegrationTest.cpp

```diff
@@ -0,0 +1,12 @@
+#include "stdafx.h"
+
+
+int main(int argc, char* argv[])
+{
+	::testing::InitGoogleTest(&argc, argv);
```

Comment:
Just as a comment, in SNOW we generally set the verbose level to be Error. Default is Warning (which includes Error ofc)

Reply:
Agree. Set verbose level to error to minimize the amount of traces